### ISSUE TITLE
Images flicker white when hover causes them to change (preloading)

### ISSUE THREAD

**Issue by Acidic9** [electron/electron#11945]
* Electron version: v1.8.2
* Operating system: Windows 10 64bit

### Expected behavior

In my window, I have a HTML button with a background image that has 3 possible urls, one for idle, one for hover and one for active.

```html
<button id="my-btn"></button>
```

```css
#my-btn {
  width: 32px;
  height: 32px;
  border: none;
  background-image: url('./img/idle.png');
}
#my-btn:hover { background-image: url('./img/hover.png') }
#my-btn:active { background-image: url('./img/down.png') }
```

![Working buttons](https://i.gyazo.com/20dab558db675ac30e9c02b59a2251ef.gif)

### Actual behavior

The image flickers white upon changing (hover/down). This is of course because the image is being loaded only once it needs to be drawn on the window.

_I tried to record a gif of the issue, but the final recording of the gif actually shows it as it's working, the flicker is only visible on my monitor, not when recorded._

I've tried adding the following HTML

```html
<div style="hidden" id="preload">
  <img src="./img/idle.png">
  <img src="./img/hover.png">
  <img src="./img/down.png">
</div>

<script>
  win.webContents.once('dom-ready', () => {          // Also tried 'ready' and 'load'
    document.getElementById('preload').remove()
  })
</script>
```

but the issue still seems to remain.

### How to reproduce

See the HTML and CSS in `Expected behavior`.

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v2.0.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
